src/app/main.c

```diff
@@ -146,6 +162,27 @@ rpmostree_option_context_parse (GOptionContext *context,
     invocation ? invocation->command->flags : RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD;
   gboolean use_daemon = ((flags & RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD) == 0);
 
+  if (invocation && invocation->command->description != NULL)
```

Comment:
Maybe let's just simplify this by adding the description as part of `option_context_new_with_commands` for "root" commands (those that have subcommands), since we're already doing string goo there. And here just do a simple `g_option_context_set_summary (context, invocation->command->description)` if there is no summary yet?